doc(viewer): update native plugins documentation (#35)
bimdata/documentation: src/viewer/reference/native_plugins.md

# Native plugins

The BIMDataViewer is shipped with native plugins that allow basic interaction with ifcs/models.

> To enabled/disabled/configure them, use their name with the corresponding configuration on the `plugins` section

Reviewer: ```suggestion
To enable/disable/configure them, use their name with the corresponding configuration on the `plugins` section
```

- name: bcf

The BCF plugin allows to interact with the BIMData BCF API by opening the BCF Manager plugin.

### Configuration

| Name        | Type     | Description                              | Default | Version |
| :---------- | :------- | :--------------------------------------- | :------ | :------ |

> | `topicGuid` | `string` | **guid** of the topic to open on startup |         | 2.0.0   |

Reviewer: I'd say 1.9.0, no?

Author: Anyway, I will remove the **Default** and **Version** columns because they are not relevant anymore.

- name: projection

> A plugin as button that sends projection type information on the `localContext.hub` when user change it through the UI.

Reviewer: ```suggestion
A plugin as button that send projection type information on the `localContext.hub` when user change it through the UI.
```

Author: Actually I think the final "**s**" is right here ([see](https://www.britannica.com/dictionary/eb/qa/third-person-singular-s-simple-present-tense))

- name: section

> A plugin as button that allows to create section planes in the `viewer3d` plugin.

Reviewer: ```suggestion
A plugin as button that allow to create section planes in the `viewer3d` plugin.
```
